src/chttpd_auth_cache.erl

```diff
@@ -95,7 +95,8 @@ get_from_cache(UserName) ->
 %% gen_server callbacks
 
 init([]) ->
-    {ok, #state{changes_pid = spawn_changes(0)}}.
+    self() ! {start_listener, 0},
```

Comment:
I don't have a lot experience, but I thought that sending a message to self on init is sort of a hack that may cause various issues like hanging and so. Is that true and safe to use here?